server/data-service/src/main/java/edu/illinois/ncsa/incore/service/data/controllers/DatasetController.java

```diff
@@ -444,7 +445,8 @@ public Dataset deleteDataset(@ApiParam(value = "Dataset Id from data service", r
             geoserverUsed = true;
         }
 
-        if (authorizer.canUserDeleteMember(this.username, datasetId, spaceRepository.getAllSpaces())) {
```

Comment:
any reason why we are not checking "delete authority" on top of the logic that the user is the owner of the dataset?

Reply:
Instead of this authority thing, that actually checks the id of the dataset is in the space list, it just checks if the logged in user is actually the owner. Otherwise, delete will fail